remo/reports/templates/edit_ng_report.jinja

```diff
@@ -207,6 +207,8 @@
       </div>
     </div>
 
+        <input type="hidden" name="timezone" id="timezone" value="">
```

Comment:
very small change: could you please align this properly so it lines up with the rest of the elements? If I counted correctly, this would be 6 spaces before the ```<input```.

Reply:
Done ..